package.json

```diff
@@ -51,5 +51,12 @@
     "yarn.lock": [
       "git rm --cached"
     ]
+  },
+  "dependencies": {
```

Comment:
If we decide we want this we'll probably want to move to `create-react-app` or `react-scripts` instead as those are the packages used by the end user.